Emby.Server.Implementations/Activity/ActivityLogEntryPoint.cs

```diff
@@ -349,18 +324,7 @@ void _userManager_UserCreated(object sender, GenericEventArgs<User> e)
             });
         }
 
-        void _subManager_SubtitlesDownloaded(object sender, SubtitleDownloadEventArgs e)
```

Comment:
I wonder why this was unused...

Reply:
Because the event doesn't exist anymore.